test/bba/node.go

```diff
@@ -0,0 +1,118 @@
+package bba
+
+import (
+	"github.com/DE-labtory/cleisthenes"
+	engine "github.com/DE-labtory/cleisthenes/bba"
+	"github.com/DE-labtory/cleisthenes/pb"
+	"github.com/DE-labtory/cleisthenes/test/mock"
+	"github.com/DE-labtory/iLogger"
+)
+
+const initialEpoch = 1
+
+type handler struct {
+	handleFunc func(msg cleisthenes.Message)
+}
+
+func newHandler(handleFunc func(cleisthenes.Message)) *handler {
+	return &handler{
+		handleFunc: handleFunc,
+	}
+}
+
+func (h *handler) ServeRequest(msg cleisthenes.Message) {
+	h.handleFunc(msg)
+}
+
+type Node struct {
+	addr       cleisthenes.Address
+	bba        *engine.BBA
+	grpcServer *cleisthenes.GrpcServer
```

Comment:
I think the server does not have to say that it is grpc.

Reply:
ok